README.md

```diff
@@ -377,270 +366,35 @@ signIn.off('ready');
 signIn.off('ready', onReady);
 ```
 
-### getTransaction
+### authClient
 
-Returns [authentication transaction information](https://developer.okta.com/docs/api/resources/authn#transaction-model) given a `stateToken`.
+Returns the underlying `@okta/okta-auth-js` object used by the widget
```

Comment:
I'd also make this reference to @okta/okta-auth-js a link to the auth js repository